modmesh/system.py

```diff
@@ -83,6 +82,19 @@ def _run_viewer(argv):
     wm.windowTitle = "Modmesh Viewer"
     wm.resize(w=1000, h=600)
     wm.show()
+
+    config = {
```

Comment:
Do not use a Python dictionary or C++ hash or tree to keep configurations.  They are slow for HPC.

There are cases that we may tolerate the slowness of a hash/tree, but I do not want to make it enter modmesh so soon.  It delivers a wrong impression to devs and users.